src/opensuse/42.3/Dockerfile

```diff
@@ -77,8 +81,6 @@ RUN zypper -n install \
     \
     zypper clean -a
 
-RUN npm install -g azure-cli
```

Comment:
Any clue why the azure-cli was present?  How confident are you there isn't a usage that has a dependency on this?

Reply:
It didn't get removed; just moved. I put it in the same place as 42.1/Dockerfile in the "used by VSO build automation" section.